### PR TITLE
Add profiles for building in windows and unix

### PR DESCRIPTION
**Pull request by aroberson**

### CODE REVIEW

**Comment by aroberson** [comment]
@gavin-scott We can add an profile id that is set in settings.xml which can then be used in the plugin configuration.  I don't know if this is a better option, but it is an option.

**Comment by aroberson** [comment]
@gavin-scott Look for External Properties in
https://maven.apache.org/guides/introduction/introduction-to-profiles.html

**Comment by aroberson** [comment]
@gavin-scott I found another way.  Will implement and let you know.

**Comment by aroberson** [comment]
@gavin-scott Could not find an easy way to configure the plugin in dne-paqx-packaging.  However, this does cleanup dne-paqx-distribution.

File: dne-paqx-packaging/pom.xml [hunk 1]
```diff
@@ -44,92 +43,193 @@
...
+    <profiles>
+        <profile>
+            <id>package-unix</id>
+            <activation>
+                <os>
+                    <family>unix</family>
+                </os>
+            </activation>
+            <build>
+                <plugins>
```
Comment (gavin-scott) [inline dne-paqx-packaging/pom.xml]: Please add comments that if this changes you have to change the windows section also. Other than that looks GTG.

**Comment by aroberson** [comment]
@gavin-scott Updated and fixed the configuration in the dne-paqx-packaging

**Comment by gavin-scott** [comment]
@aroberson looks like you'll have to rebase this, then it can be merged.

**Pull request merged**